libs/dh/admin/feature-user-roles/src/lib/drawer/dh-role-drawer.component.html

```diff
@@ -32,7 +32,7 @@ <h2>{{ userRoleWithPermissions.name }}</h2>
       </watt-drawer-heading>
     }
 
-    @if (basicUserRole?.status !== "Inactive") {
```

Comment:
Suggestion: Use UserRole Status Enum in the template, by binding as a public field in the component